src/Validation/Validation.php

```diff
@@ -216,6 +216,26 @@ public static function cc($check, $type = 'fast', $deep = false, $regex = null)
     }
 
     /**
+     * Used to check the count of a given value of type string, int, or array.
+     *
+     * If a string value is passed the string length is used as count.
+     *
+     * @param array|int|string $check1 The value to check the count on.
+     * @param string $operator Can be either a word or operand
+     *    is greater >, is less <, greater or equal >=
+     *    less or equal <=, is less <, equal to ==, not equal !=
+     * @param int $expectedCount The expected count value.
+     * @return bool Success
+     */
+    public static function numElements($check1, $operator, $expectedCount)
+    {
+        if (!is_array($check1) || $check1 instanceof \Countable) {
```

Comment:
`!` is needed before `$check1 instanceof \Countable`. Adding `ArrayObject` test might be nice.